packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -68,10 +68,10 @@ export const WfoStartPage = () => {
         headerStatus: SummaryCardStatus.Neutral,
         listTitle: t('activeSubscriptions.listTitle'),
         listItems:
-            subscriptionsSummaryResult?.subscriptions.page.map(
+            subscriptionsSummaryResult?.subscriptions?.page.map(
```

Comment:
As far as i know is `subscriptions` not "possibly undefined". Same goes for the other cases in this PR.